app/services/coupons/update_service.rb

```diff
@@ -60,6 +82,48 @@ def update_from_api(organization:, code:, params:)
 
     private
 
+    attr_reader :coupon, :limitations
+
+    def plan_identifiers
```

Comment:
Same comment as in create service about verbosity :smile:

Reply:
fixed :)